app/products/calls/connection/foreground_service.ts

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import VIForegroundService from '@voximplant/react-native-foreground-service';
+
+import {logError} from '@utils/log';
+
+const channelConfig = {
+    id: 'calls_channel',
+    name: 'Mattermost',
+    description: 'Mattermost Calls microphone while app is in the background',
+    enableVibration: false,
+};
+
+// Note: multiple calls with same arguments are a noop.
+export const foregroundServiceSetup = () => {
+    VIForegroundService.getInstance().createNotificationChannel(channelConfig);
+};
+
+export const foregroundServiceStart = async () => {
+    const notificationConfig = {
+        channelId: 'calls_channel',
+        id: 345678,
```

Comment:
Should there be some logic behind the id value? Should it be a constant somewhere?

Reply:
It's completely arbitrary (just has to be unique for the channel), and since we're not using this outside the file, or need to change it, 1/5 I don't think we need a constant.